### DISCUSSION TITLE
关于饰品提取和模拟宇宙功能的疑问

### DISCUSSION THREAD

**Discussion by Phlogist** [LmeSzinc/StarRailCopilot#583] [Q&A]
如题，我之前没有开过这两个功能，但是现在主刷饰品了，要开饰品提取。

在饰品提取设置中，选项”使用开拓力刷取内圈仪器选项“下面的优先级说明是”优先于模拟宇宙和养成规划“，那我的问题是，这个选项的优先级是否优先于每日副本？

**Comment by LmeSzinc** [comment]
优先于普通仪器
